docs/docs/functions/macros.md

```diff
@@ -70,23 +93,74 @@ Defines a new macro.
 
 ## defnode
 
-Defines a new node on which chains can be scheduled.
+Defines a new node on which chains can be scheduled (to run).
+
+A node is a self-contained execution-context and software environment (like a server) that executes the chain code logic. It can run both on the local hardware as well as  peer-to-peer hardware over the network (blockchain). 
```

Comment:
Nit: one too many space between `as` and `peer-to-peer`.
```suggestion
A node is a self-contained execution-context and software environment (like a server) that executes the chain code logic. It can run both on the local hardware as well as peer-to-peer hardware over the network (blockchain). 
```